Frontend/ui-customer/src/Screens/OnBoardingFlow/AccountSetUpScreen/Controller.purs

```diff
@@ -67,7 +76,13 @@ data Action
   | NameEditTextActionController PrimaryEditTextController.Action
   | GenericHeaderActionController GenericHeaderController.Action
   | PopUpModalAction PopUpModal.Action
+  | MenuButtonAC MenuButtonController.Action
```

Comment:
we are not using menu button in this screen

Reply:
removed.